main.tex

```diff
@@ -78,15 +82,17 @@ \section{Introduction}\label{sec:introduction}
 recent variants such as OmegaTFT, Zero determinant and other memory one
 strategies, strategies based on finite state machines, lookup tables, neural
 networks, and other machine learning based strategies, and a collection of novel
-strategies. Not all strategies have been considered for this study: same use
-knowledge of the game such as the number of turns and others have a high
+strategies. Not all strategies have been considered for this study: excluded
+are those that make use of knowledge of the game such as the number of turns
```

Comment:
This is a bit misleading, all our ZD strategies make use of the values of the game right? The only ones we've removed are the ones that make use of the length of the game.